copilot/environments/overrides/cfn.patches.yml

```diff
@@ -40,3 +40,6 @@
   path: /Resources/EnvironmentHTTPSSecurityGroupIngressFromPublicALB
 - op: remove
   path: /Resources/EnvironmentSecurityGroupIngressFromSelf
+- op: add
+  path: /Resources/PublicLoadBalancer/Properties/Name
+  value: !Sub ${EnvironmentName}-${AppName}-pub-lb
```

Comment:
Okay, this is much easier than expected!

Reply:
Ya, it's surprisingly ... ok.